tests/hubspot/test_hubspot_pipeline.py

```diff
@@ -0,0 +1,86 @@
+from unittest.mock import patch
+
+import pytest
+import requests
+
+from pipelines.hubspot import fetch_data, CRM_CONTACTS_ENDPOINT, CRM_COMPANIES_ENDPOINT, CRM_DEALS_ENDPOINT, CRM_PRODUCTS_ENDPOINT, CRM_TICKETS_ENDPOINT, CRM_QUOTES_ENDPOINT
+from tests.hubspot.mock_data import mock_contacts_data, mock_companies_data, mock_deals_data, mock_products_data, mock_tickets_data, mock_quotes_data
+
+
+@pytest.fixture()
+def mock_response():
```

Comment:
thanks for writing the tests with mocked data! besides this test could you add end to end test like the one here:
https://github.com/dlt-hub/pipelines/blob/master/tests/pipedrive/test_pipedrive_pipeline.py#L11
it is just a few lines

hubspot gives you dev accounts with test data. just ping me for a key. I'll add it to our CI as well